sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs

```diff
@@ -56,6 +49,7 @@ public sealed class EventHubScope : IAsyncDisposable
         ///   Initializes a new instance of the <see cref="EventHubScope"/> class.
         /// </summary>
         ///
+        /// <param name="eventHubResourceId">The ARM resource identifier of the Event Hub that was created.</param>
```

Comment:
I don't see this parameter in the method below - did you mean to add it, or is this param comment supposed to be somewhere else?

Reply:
Good catch.   I pulled it after learning the "you can create resource identifiers with a static method" trick from Michael!